server/datastore/mysql/migrations/tables/20230405232025_AddBootstrapPackagesTable.go

```diff
@@ -0,0 +1,32 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230405232025, Down_20230405232025)
+}
+
+func Up_20230405232025(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+          CREATE TABLE mdm_apple_bootstrap_packages (
+            team_id    int(10) unsigned NOT NULL,
+            name       varchar(255),
+            sha256     BINARY(32) NOT NULL,
+            bytes      longblob,
```

Comment:
Should we document max size somewhere? (depends on MySQL configuration)

Reply:
good point, I have a follow up PR in the works with documentation (including the API endpoint docs missing from this PR) I will include that there.